capstone/src/main/java/com/google/sps/data/ReadData.java

```diff
@@ -85,75 +103,63 @@ public void readFile(FileReader file, String zipCode) {
 
         final int START_INDEX = 11;
 
-        int indexOfLong = (values.length - EXPECTED_LENGTH) + START_INDEX + 1;
+        int indexOfInt = (values.length - EXPECTED_LENGTH) + START_INDEX + 1;
 
-        for (int i = START_INDEX; i < indexOfLong; i++) {
+        for (int i = START_INDEX; i < indexOfInt; i++) {
           text += values[i];
         }
 
         float score = 0;
 
-        if (text.length() > 0) {
+        if (sentimentService != null && text.length() > 0) {
           score = sentimentService.getSentiment(text);
         }
 
-        long responseTimeOne = Long.parseLong(values[indexOfLong++]);
-        long responseTimeTwo = Long.parseLong(values[indexOfLong++]);
-        long responseTimeThree = Long.parseLong(values[indexOfLong]);
+        int responseTimeOne = (int) Math.round(Double.parseDouble(values[indexOfInt++]));
```

Comment:
Why did you end up using ints here instead of doubles?

Reply:
Doubles and floats were giving precision errors when stored in data store, so I stored them as ints. For example, if i wanted to store 217.48 I would store it as 21748 and then divide it by 100 when fetching.

Reply:
+1 doubles and floats give precision errors. This is an accepted practice but just to be clear, you may want to rename the variable to `responseTimeOneCentis` if that's easy to do. Either that or leave documentation indicating that we store in the hundreds.